src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -25,13 +26,13 @@ public CinemaHallController(CinemaHallService cinemaHallService,
         this.cinemaHallMapper = cinemaHallMapper;
     }
 
-    @PostMapping
-    public CinemaHallResponseDto add(@RequestBody CinemaHallRequestDto requestDto) {
+    @PostMapping("/add")
+    public CinemaHallResponseDto add(@RequestBody @Valid CinemaHallRequestDto requestDto) {
         CinemaHall cinemaHall = cinemaHallService.add(cinemaHallMapper.mapToModel(requestDto));
         return cinemaHallMapper.mapToDto(cinemaHall);
     }
 
-    @GetMapping
+    @GetMapping("/getAll")
```

Comment:
Why did you add  @PostMapping("/add") and  @GetMapping("/getAll") to your URL?

Reply:
For  testing the code